finesse/hardware/plugins/spectrometer/ftsw500_interface.py

```diff
@@ -70,135 +60,141 @@ def __init__(self) -> None:
         """Create a new FTSW500Interface."""
         super().__init__()
 
-        self._requester = socket(AF_INET, SOCK_STREAM)
-        self._requester.settimeout(FTSW500_TIMEOUT)
-
-        self._requester.connect((FTSW500_HOST, FTSW500_PORT))
-
-        self._status = SpectrometerStatus.UNDEFINED
+        sock = socket(AF_INET, SOCK_STREAM)
+        sock.settimeout(FTSW500_TIMEOUT)
+        sock.connect((FTSW500_HOST, FTSW500_PORT))
+        self._socket = sock
 
+        # Timer to poll status
         self._status_timer = QTimer()
-        self._status_timer.timeout.connect(self._request_status)
+        self._status_timer.timeout.connect(self.pubsub_errors(self._update_status))
         self._status_timer.setInterval(int(FTSW500_POLLING_INTERVAL * 1000))
         self._status_timer.setSingleShot(True)
 
-        self._request_status()
-
-    def _set_output_path_prefix(self, output_path_prefix: str) -> None:
-        """Set the output path prefix for FTSW500 data."""
-        self._requester.sendall(
-            f"setOutputPathPrefixField&{output_path_prefix}\n".encode()
-        )
-        self._requester.recv(1024)
-
-    def _set_output_prefix(self, output_prefix: str) -> None:
-        """Set the output prefix for FTSW500 data."""
-        self._requester.sendall(f"setOutputPrefixOnlyField&{output_prefix}\n".encode())
-        self._requester.recv(1024)
-
-    def set_data_description(self, data_description: str) -> None:
-        """Set the description for FTSW500 data."""
-        self._requester.sendall(
-            f"setDataDescriptionField&{data_description}\n".encode()
-        )
-        self._requester.recv(1024)
```

Comment:
These will be required for #534

Reply:
Seeing as they needed tweaking anyway, I thought I'd just remove them for now and add them back in when we were going to use them.